sunpycube/cube/datacube.py

```diff
@@ -639,3 +640,13 @@ def plot_wavelength_slice(self, offset, **kwargs):
             offset, cumul_cube_lengths)
         plot = self[sequence_index].plot_wavelength_slice(cube_index, **kwargs)
         return plot
+
+
+class Slice_helper(object):
+    """Helper class to make slicing in index_as_cube more pythonic"""
+
+    def __init__(self, seq):
+        self.seq = seq
+
+    def __getitem__(self, item):
```

Comment:
Either here or in one of the docstrings above can you list the different allowed types that item can take?